package.json

```diff
@@ -17,9 +17,9 @@
   },
   "homepage": "https://github.com/scality/utapi#readme",
   "dependencies": {
-    "arsenal": "scality/Arsenal#67365083",
+    "arsenal": "scality/Arsenal#121880a",
```

Comment:
I'm thinking we can leave the Arsenal dependency as-is because utapi creates its own Redis client, and does not import it from Arsenal.